docs/intro/types.md

```diff
@@ -48,6 +48,7 @@ Result:
 
 #### Agtype NULL vs Postgres NULL
 
+The concept is same for Agtype NULL and Postgres NULL as it is for Cypher.

```

Comment:
The concept of NULL in Agtype (and Cypher) is the same as it is in Postgres.

Reply:
Actually, in the above paragraph ("NULL" heading) the information about Cypher NULL is given. The NULL in PostgreSQL and Agtype works the same way as it works in Cypher. So that is why, I am taking information given for Cypher NULL as reference point to convey that NULL works the same way in PostgreSQL and Agtype as well.
We can also structure it as 'The concept of NULL in Agtype and Postgres is same as it is in Cypher.'